definitions.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+# Path to the root of the project
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+

```

Comment:
We can either move the contents of the file to `import_vars` only or if you like to keep the constants in a separate file, we can create a submodule.

Let me know if you need any help with that.

Reply:
the idea was that this file defines where the root of the project is, because if i tried to define the root in a nested file it would be dependent on the path of the file I guess

Reply:
We can use Pathlib(https://docs.python.org/3/library/pathlib.html) to resolve the path. And I feel that we can move these constants and env_populator to a module of their own as the constants are only being used in the `env_populator.py`.